Taskfile.yml

```diff
@@ -25,3 +46,40 @@ tasks:
         yq -i ".dockers[0].extra_files |= [\"$replacement\"]" ci/goreleaser/goreleaser-el7.yml
         yq -i ".dockers[4].extra_files |= [\"$replacement\"]" ci/goreleaser/goreleaser.yml
       - echo goreleaser release --clean --snapshot -f ci/goreleaser/goreleaser.yml
+
+  test:opentelemetry:setup:
+    desc: "setup e2e opentelemetry tests"
+    deps: 
+      - deps:tracetest
+      - docker
+    status:
+      - tracetest version
+    cmds:
+      - docker compose -f ci/tests/tracing/docker-compose.yml up -d
+      - tracetest configure -g --endpoint http://localhost:11633
+
+
+  test:opentelemetry:test:
```

Comment:
Drop the test prefix and move opentelemetry into .taskfiles/otel/Taskfile to drop the opentelemetry prefix from yaml. Include otel taskfile into root taskfile

Reply:
@titpetric there's a dependency between the setup step and the `docker` task. Any idea on how can we solve that?

Reply:
@tbuchaillot drop the dependency, but keep the docker target. No suggestions atm